src/server/routes/entity/entity.tsx

```diff
@@ -150,9 +150,9 @@ export function displayEntity(req: PassportRequest, res: $Response) {
 		if (EntityComponent) {
 			const props = generateProps(req, res, {
 				alert,
-				identifierTypes
+				identifierTypes,
+				genderOptions: res.locals.genders
```

Comment:
My linter is complaining that the keys are not in alphabetical order. Not that I care that much, but…
```suggestion
				genderOptions: res.locals.genders,
				identifierTypes
```